rhcos-layer/Makefile

```diff
@@ -0,0 +1,72 @@
+# Default variables
+
+REGISTRY_AUTH_FILE := ocp_pull_secret.json
+IMAGE_TDX ?= quay.io/bpradipt/rhcos-layer/ocp-4.18:latest-tdx
+IMAGE_SNP ?= quay.io/bpradipt/rhcos-layer/ocp-4.18:latest-snp
+
+TEE ?= tdx
+
+# Default target
+.PHONY: build
+build: #check-podman get-pull-secret get-ocp-release-image ## Default target: Build the RHCOS layer
+	@echo "Building RHCOS layer..."
+	if [ "$(TEE)" == "tdx" ]; then \
```

Comment:
It seems we need '=' instead of '==' for /bin/sh to avoid warning 'unexpected operator'.

Reply:
What if you run with latest bash ?

Reply:
I have updated it